### ISSUE TITLE
HttpAuth.php and Httpauth.php

### ISSUE THREAD

**Issue by lodgic** [Intervention/httpauth#8]
Is there a reason for having these two files with a case difference? They have the same namespace `Intervention\HttpAuth` and class name `HttpAuth`.

**Comment by olivervogel** [comment]
Did you run into any problems or why are you asking? I renamed the files a few days ago but for some reason the old files were still there.

I removed the following files:
```
'src/Httpauth.php'
'src/Laravel/Facades/Httpauth.php'
'src/Laravel/HttpauthServiceProvider.php'
```

**Comment by lodgic** [comment]
My IDE was complaining that it found multiple references to that class. Then I saw the duplication. 

Functionality wasn’t affected as far as I could tell.

**Comment by olivervogel** [comment]
Should be fixed now in 3.0.1

**Comment by lodgic** [comment]
Kudo’s on the quick resolution. 😉
